tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +122,162 @@ module('Acceptance | guid file | registration files', hooks => {
+                .containsText(funder.award_uri, `Funder award URI is unchanged for ${funder.funder_name}.`);
+            // Award number
+            assert.dom('[data-test-target-funder-award-number-label]').hasText('Award number',
+                'Funder award number label is properly set.');
+            assert.dom(`[data-test-target-funder-award-number="${funder.funder_name}"]`)
+                .hasText(funder.award_number, `Funder award number is unchanged for ${funder.funder_name}`);
+        }
+        // Target title
+        assert.dom('[data-test-target-title-label]').hasText('Title',
+            'File metadata target title text is properly set.');
+        assert.dom('[data-test-target-title]').exists();
+        // Target description
+        assert.dom('[data-test-target-description-label]').hasText('Description',
+            'File metadata target description text is properly set.');
+        assert.dom('[data-test-target-description]').exists();
+    });
+
+    // Node type
+    test('Node type', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        assert.dom('[data-test-metadata-node]').hasText('Registration Metadata',
+            'Node noun for registration properly set.');
+    });
+
+    // Save and cancel buttons with write permissions
+    test('Save and cancel', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify download
+        assert.dom('[data-test-download-button]').exists();
+
+        // Verify edit form
+        await click('[data-test-edit-metadata-button]');
+        assert.dom('[data-test-title-field]').exists();
+        assert.dom('[data-test-description-field]').exists();
+        assert.dom('[data-test-select-resource-type]').exists();
+        assert.dom('[data-test-select-resource-language]').exists();
+        assert.dom('[data-test-cancel-editing-metadata-button]').exists();
+        assert.dom('[data-test-save-metadata-button]').exists();
+        await click('[data-test-cancel-editing-metadata-button]');
+        assert.dom('[data-test-edit-metadata-form]').doesNotExist();
+        await click('[data-test-edit-metadata-button]');
+
+        // Screenshot before changes
+        await percySnapshot(assert);
+        // Update title
+        const editTitleTextArea = document.querySelectorAll('textarea')[0];
+        editTitleTextArea.textContent = 'A test title.';
+        assert.dom('[data-test-title-field]')
+            .containsText('A test title.', 'Metadata title field updates.');
+        // Update description
+        const editDescriptionTextArea = document.querySelectorAll('textarea')[1];
+        editDescriptionTextArea.textContent = 'A test description.';
```

Comment:
Same with this. You should be able to simplify it with `fillIn`.

Reply:
If fillIn is able to work, I can update this.

Reply:
We spoke about this briefly where the query selector for the title and description text areas resulted in a changed textarea for value but not innerText or textContent. The versions tried had not worked and this is currently being further tested. Until it passes, I can keep these changes until we can resolve why the update is not occurring with fillIn but the full JS query selector is currently working.

Reply:
Same here.

Reply:
@adlius The comment above should answer this.